packages/ia-components/integration/audio_player.test.js

```diff
@@ -0,0 +1,98 @@
+import { Builder, By, until } from 'selenium-webdriver';
+import chrome from 'selenium-webdriver/chrome';
+import driverTraversalDecorator from './utils/traversal';
+
+const URL = 'http://localhost:9001/?selectedKind=Theatres&selectedStory=Audio%20with%20YouTube%2C%20Spotify&full=0&addons=1&stories=1&panelRight=0&addonPanel=%40storybook%2Faddon-a11y%2Fpanel';
+
+const selectors = {
+  frame: '#storybook-preview-iframe',
+  theater_container: '.theatre__wrap',
+  cover_image: '.background-photo',
+  track1_button: '.audio-track-list button[data-track-number="1"]',
+  track_no: '.track-number',
+  track_name: '.track-title',
+  track_duration: '.track-length',
+  liner_notes: '#IABookReaderWrapper'
+};
+
+describe('Audio Player', () => {
+  let driver;
+
+  let to, asyncTo, toFrame, toParentFrame;
```

Comment:
SUGGESTION: For AirBNB JS style, one declaration per line: https://github.com/airbnb/javascript#variables--one-const